youtube_dl/extractor/keezmovies.py

```diff
@@ -105,7 +105,8 @@ def extract_format(format_url, height=None):
                 raise ExtractorError(
                     'Video %s is no longer available' % video_id, expected=True)
 
-        self._sort_formats(formats)
+        if len(formats) > 0:
+            self._sort_formats(formats)
```

Comment:
No.

Reply:
@dstftw - Please elaborate?  This will allow the `extract_format` call to complete cleanly in the event that no formats can be detected, so that the generic extractor can be picked up later.  Thanks!

Reply:
This method is used in another extractors. **Do not change the default behavior**.

Reply:
Ah, you are correct!  My apologies, I completely neglected to check that.  I will make the necessary corrections.